docs/content/direct/hub-chart.md

```diff
@@ -0,0 +1,157 @@
+export KUBESTELLAR_VERSION=0.23.0-alpha.1
+```
+
+## Pre-requisites
+
+A list of executable that may be required during the deployment can be fund [here](./pre-reqs.md).
+At minimum [Helm](https://helm.sh/) is required.
+
+The setup of KubStellar via the Hub chart requires the existance of at least one cluster
+to be used for the deployment of the chart.
+
+This can be:
+
+1. A local **Kind** cluster with an ingress with SSL passthrough and a mapping to host port 9443
+
+This option is particularly useful for first time users or users that would like to have a local deployment.
+
+For convenience, a new local Kind clusters that satisfies the requirements for KubeStellar setup
+and that can be used to exercises the [examples](./examples.md) can be created with the following command:
+
+```shell
+sudo su <<EOF
+bash <(curl -s https://raw.githubusercontent.com/kubestellar/kubestellar/v${KUBESTELLAR_VERSION}/scripts/create-kind-cluster-with-SSL-passthrough.sh) --name kubestellar --port 9443
+EOF
+```
+
+It is important to note that, if a different name is used for the Kind cluster, then its control plane name must be also be referenced during the Helm chart installation by setting `--set "kubeflex-controller.hostContainer=<control-plane-name>".
+
+2. A local **k3s** cluster with an ingress with SSL passthrough and a mapping to host port 9443
+
+... Under onstruction ...
+
+3. An **OpenShift** cluster
+
+When using this option, one is required to explicitely set the `isOpenShift` variable to `true` by including `--set "kubeflex-operator.isOpenShift=true"` in the Helm chart installation command.
+
+
+## KubeStellar Hub Chart values
+
+The KubeStellar chart makes available to the user several values that may be used to customize its installation into an existing cluster:
+
+```yaml
+# KubeFlex override values
+kubeflex-operator:
+  install: true # enable/disable the installation of KubeFlex by the chart (default: true)
+  installPostgreSQL: true # enable/disable the installation of the appropriate version of PostgreSQL required by KubeFlex (default: true)
+  isOpenShift: false # set this variable to true when installing the chart in an OpenShift cluster (default: false)
+  # Kind cluster specific settings:
+  domain: localtest.me # used to set the Control Planes DNS in a Kind cluster installation (default: localtest.me)
+  externalPort: 9443 # used to set the port to access the Control Planes API (default: 944)
+  hostContainer: kubeflex-control-plane # used to set the name of cluster control plane (default: kubeflex-control-plane, which corresponds to a Kind cluster with name kubeflex)
+
+# Determine if the Post Create Hooks should be installed by the chart
+InstallPCHs: true
+
+# List the Inventory and Transport Spaces (ITSes) to be created by the chart
+# Each ITS consists of a mandatory unique name and an optional type, which could be either host|vcluster (default to vcluster, if not specified)
+ITSes: # ==> installs ocm + ocm-status-addon

```

Comment:
Is it even valid YAML to follow `foo:` with nothing? I think I have gotten blowback from this before. If you mean that the default is an empty JSON array then I would explicitly write `[]` here.

Reply:
I can tell you that it's is valid, for Helm

The `==>` applies to all but I will change it to be more clear

OCM is removed